aws-api/src/main/java/com/amplifyframework/api/aws/AppSyncGraphQLRequestFactory.java

```diff
@@ -160,20 +186,29 @@ private AppSyncGraphQLRequestFactory() { }
                     new GsonVariablesSerializer()
             );
         } catch (AmplifyException exception) {
-            throw new ApiException(
+            throw new IllegalStateException(
                     "Could not generate a schema for the specified class",
-                    exception,
-                    "Check the included exception for more details"
+                    exception
             );
         }
     }
 
+    /**
```

Comment:
Instead of all of these, you can do:

```
@SuppressWarnings("checkstyle:MissingJavadocMethod")
```

On top of the class definition. This will disable the check for the entire class.

Of rouse, better would be to write the docs. But, the suppression is still better than the not-filled-in stuff, I think.

Reply:
I added the docs. Most of them are very similar to the ones in `ModelQuery`/`ModelMutation`/`ModelSubscription`